xskillscore/core/resampling.py

```diff
@@ -145,8 +145,11 @@ def resample_iterations_idx(
 
     Examples
     --------
-    >>> a = xr.DataArray(np.random.rand(1000, 3, 3), dims=['time', 'x', 'y'])
-    >>> xs.resample_iterations_idx(a, 500, 'time') # doctest: +ELLIPSIS
+    >>> a = xr.DataArray(np.random.rand(1000, 3, 3),
+    ...                  coords=[("time", np.arange(1000)),
+    ...                          ("x", np.arange(3)),
+    ...                          ("y", np.arange(3))])
+    >>> xs.resample_iterations_idx(a, 500, 'time') # doctest: +SKIP
```

Comment:
```suggestion
    >>> xs.resample_iterations_idx(a, 500, 'time')
```

Reply:
Shouldn’t this work?

Reply:
Not worth it. The output is huge

Reply:
![Screenshot from 2021-05-05 14-13-52](https://user-images.githubusercontent.com/17162724/117189437-3334a380-adac-11eb-9636-c7ea4eea892c.png)